aws/resource_aws_ebs_volume.go

```diff
@@ -286,6 +298,7 @@ func resourceAwsEbsVolumeRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("snapshot_id", aws.StringValue(volume.SnapshotId))
 	d.Set("outpost_arn", aws.StringValue(volume.OutpostArn))
 	d.Set("multi_attach_enabled", volume.MultiAttachEnabled)
+	d.Set("throughput", aws.Int64Value(volume.Throughput))
```

Comment:
to avoid having to configure it when using `gp3`, should we default it to `125` if the `VolumeType === gp3`? 🤔

Reply:
The [API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html#API_CreateVolume_RequestParameters) specifies no default although you are correct that if it's not specified for `gp3` volumes the `DescribeVolumes` call returns a value of `125`.
The attribute is declared as `Computed: true` to handle this.